sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusScaleMonitor.cs

```diff
@@ -163,6 +163,11 @@ internal static ServiceBusTriggerMetrics CreateTriggerMetrics(ServiceBusReceived
             };
         }
 
+        private static bool MessageWasNotScheduled(ServiceBusReceivedMessage message)
+        {
+            return message.ScheduledEnqueueTime == DateTimeOffset.MinValue;
```

Comment:
FYI we should be able to use the new message state property once we add it to the T2 library. T1 PR - https://github.com/Azure/azure-sdk-for-net/pull/25158/files

Reply:
So as it stands, this code would only handle scheduled messages, not deferred? I.e. if the first message in the queue was deferred and we peek it, we still have the same issue?

Reply:
@JoshLove-msft I gave the message state for T2 a go https://github.com/Azure/azure-sdk-for-net/pull/25446

Reply:
I pushed code that gets out the annotated message until we have the SDK that supports the simple property